test/test-udp-send-unreachable.c

```diff
@@ -138,12 +155,45 @@ TEST_IMPL(udp_send_unreachable) {
                   send_cb);
   ASSERT(r == 0);
 
+  r = uv_udp_init(uv_default_loop(), &client2);
+  ASSERT(r == 0);
+
+  r = uv_udp_bind(&client2, (const struct sockaddr*) &addr3,
+		  UV_UDP_LINUX_RECVERR);
```

Comment:
Also align arguments here (usually 1 per line)

Reply:
Ok.

Just a suggestion, I am not arguing here - what we did in BIND 9 was that we wrote close enough clang-format rules and went with that. Now instead of having formatting as part of the review, we have automated rule that fails CI if somebody forgets to reformat the new code (there are hooks for most of the common editors and git-clang-format tool to help with workflow). This is something that you might want to consider as a group because it is very helpful.

Reply:
Ok, added a note there, is that enough?

Reply:
Hopefully, now everything is aligned.